microprofile-open-api/src/test/java/org/jboss/eap/qe/microprofile/openapi/model/ProgrammingModelTest.java

```diff
@@ -0,0 +1,134 @@
+public class ProgrammingModelTest {
+
+    private final static String DEPLOYMENT_NAME = ProgrammingModelTest.class.getSimpleName();
+
+    private static String openApiUrl;
+    private static OnlineManagementClient onlineManagementClient;
+
+    @BeforeClass
+    public static void setup() throws ManagementClientRelatedException, ConfigurationException {
+        openApiUrl = OpenApiDeploymentUrlProvider.composeDefaultOpenApiUrl();
+        //  MP OpenAPI up
+        onlineManagementClient = ManagementClientProvider.onlineStandalone();
+        if (!OpenApiServerConfiguration.openapiSubsystemExists(onlineManagementClient)) {
+            OpenApiServerConfiguration.enableOpenApi(onlineManagementClient);
+        }
+    }
+
+    @AfterClass
+    public static void tearDown() throws ManagementClientRelatedException, IOException {
+        //  MP OpenAPI down
+        try {
+            OpenApiServerConfiguration.disableOpenApi(onlineManagementClient);
+        } finally {
+            onlineManagementClient.close();
+        }
+    }
+
+    @Deployment(testable = false)
+    public static WebArchive createCentralDeployment() {
+        WebArchive deployment = ShrinkWrap.create(
+                WebArchive.class,
+                String.format("%s.war", DEPLOYMENT_NAME))
+                .addClasses(RestApplication.class)
+                .addClasses(
+                        District.class,
+                        DistrictEntity.class,
+                        DistrictService.class,
+                        InMemoryDistrictService.class,
+                        DistrictsResource.class,
+                        RoutingServiceConstants.class)
+                .addClasses(OpenApiModelReader.class, OpenApiFilter.class)
+                .addAsResource(
+                        "META-INF/microprofile-config.properties")
+                .addAsWebInfResource(EmptyAsset.INSTANCE, "beans.xml");
+        return deployment;
+    }
+
+    /**
+     * @tpTestDetails Tests for application endpoint to be working
+     * @tpPassCrit Verifies that the returned JSON data is a non empty list of districts
+     * @tpSince EAP 7.4.0.CD19
+     */
+    @Test
+    @InSequence(1)
+    public void testAppEndpoint() {
+        RestAssured.when()
+                .get(OpenApiDeploymentUrlProvider.composeDefaultDeploymentBaseUrl(DEPLOYMENT_NAME + "/districts/all"))
+                .then()
+                .statusCode(200)
+                .contentType(Matchers.equalToIgnoringCase(MediaType.APPLICATION_JSON))
+                .body(Matchers.not(Matchers.empty()));
+    }
+
+    /**
+     * @tpTestDetails Verifies MP OpenAPI programming model by creating custom base document
+     * @tpPassCrit Verifies that {@link OpenAPI#getInfo()} has been successfully modified by {@link OpenApiModelReader}
+     * @tpSince EAP 7.4.0.CD19
+     */
+    @Test
+    @InSequence(2)
+    public void testOpenApiDocumentForOpenApiInfoChange() {
+        RestAssured.when().get(openApiUrl)
```

Comment:
`RestAssured.when().get` could you put it into static import in all occurrences, please?

Reply:
Done. Also managed to do the same for `org.hamcrest.Matchers`, thanks for spotting this.